src/views/Incorporation/IncorporationReviewConfirm.vue

```diff
@@ -314,6 +314,15 @@ export default class IncorporationReviewConfirm extends Vue {
   get isCourtOrderInvalid (): boolean {
     return (this.getValidateSteps && !this.getCourtOrderStep.valid)
   }
+
+  /** Is true when the role is user, set the court order component to valid */
+  get isRoleUser (): boolean {
+    if (!this.isRoleStaff) {
+      this.setCourtOrderValidity(true)
+      return true
+    }
+    return false
+  }
```

Comment:
I'm sure this works but there's a better, common way.

See https://github.com/bcgov/business-create-ui/blob/4d4a70be574b8cca57a76fe3a2aadac5466e02b3/src/store/store.ts#L746 and line 730.

Reply:
Thank you. I just updated store.js to fix this issue.